core/src/main/java/org/mobilitydata/gtfsvalidator/notice/PointNearPoleNotice.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.notice;
+
+import com.google.common.collect.ImmutableMap;
+
+/**
+ * A point is too close to the North or South Pole.
```

Comment:
Same here: how do you define "too close"? Would it be worth indicating the distance to the poles in this notice by adding a new field `distanceToPole`?

Reply:
Reading the code in the subsequent files I suggest the following change to the docs:

```suggestion
 * A point is less than @code{1.0} (unit to be detailed) from the North or South Pole.
```